AdminClient/src/app/auth/auth.page.ts

```diff
@@ -0,0 +1,168 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { LoadingController, MenuController } from '@ionic/angular';
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { AppState } from '../reducers';
+import * as AuthActions from './store/auth.actions';
+import { AuthState } from './store/auth.reducer';
+
+@Component({
+  selector: 'app-auth',
+  styleUrls: ['./auth.page.scss'],
+  templateUrl: './auth.page.html'
+})
+
+export class AuthPage implements OnInit {
+
+  /**
+   * The authentication form
+   */
+  protected authForm: FormGroup;
+
+  /**
+   * Holds all the subscriptions that will need to be cleaned up when a view swaps
+   */
+  private subscriptions = new Subscription();
+
+  /**
+   * This is the loader that was created and currently exists on the page
+   * This variable will be used to dismiss that existing loader
+   */
+  private pageLoader: HTMLIonLoadingElement;
+
+  constructor(
+    public menuController: MenuController,
+    public loadingController: LoadingController,
+    private store: Store<AppState>,
+    private snackBar: MatSnackBar,
+  ) {
+
+  }
+
+  ionViewWillEnter(): void {
```

Comment:
just curious, what's the reasoning behind using ionic for web here?

Reply:
Copy and Pasted to have the tests pass.

This is being removed in #34